### ISSUE TITLE
ugrep -G option disables the use of -e (--regex)

### ISSUE THREAD

**Issue by andresroldan** [Genivia/ugrep#105]
On traditional grep, the -e flag works by searching multiple regexes. On ugrep, when the -G is passed, the -e parameters won't work:

```bash
aroldan@volatile:~$ cat test.txt 
hello
world
aroldan@volatile:~$ ugrep -e hello -e world test.txt
hello
world
aroldan@volatile:~$ ugrep -G -e hello -e world test.txt
aroldan@volatile:~$ ugrep -G -e hello test.txt
hello
aroldan@volatile:~$ /bin/grep -e hello -e world test.txt
hello
world
aroldan@volatile:~$ 
```

This is my version information:

```bash
aroldan@volatile:~$ ugrep --version
ugrep 3.1.7 x86_64-pc-linux-gnu +sse2 +pcre2_jit +zlib +bzip2 +lzma +lz4
License BSD-3-Clause: <https://opensource.org/licenses/BSD-3-Clause>
Written by Robert van Engelen and others: <https://github.com/Genivia/ugrep>
```

**Comment by genivia-inc** [comment]
Will fix this asap.

**Comment by genivia-inc** [comment]
Fixed in 3.1.8.
